tokio/src/sync/tests/loom_rwlock.rs

```diff
@@ -76,3 +76,24 @@ fn concurrent_read_write() {
         assert_eq!(10, *guard);
     });
 }
+#[test]
+fn downgrade() {
```

Comment:
It does not look like `loom_rwlock.rs` is being included [here](https://github.com/tokio-rs/tokio/blob/30b40ef518d1767736c8ff47c794481e88bfd3e9/tokio/src/sync/tests/mod.rs).

Reply:
It is not. I was not sure whether this is intentional or not. Will include